nservicebus/rabbitmq/connection-settings_tls-settings_rabbit_[4,].partial.md

```diff
@@ -0,0 +1,15 @@
+### UseTls
+
+Indicates if the connection to the broker should be secured with [TLS](#transport-layer-security-support).
+
+Default: `false`
+
+
+### CertPath
```

Comment:
I discussed this with @SimonCropp: we may need to flesh out the relationship of these properties a little. E.g. if this is set, then X must also be set. If Y is not set then setting this will fail, or will be redundant etc.